lib/radix.c

```diff
@@ -243,10 +243,10 @@ rn_satsifies_leaf(char *trial, register struct squid_radix_node *leaf, int skip)
 
 struct squid_radix_node *
 squid_rn_match(void *v_arg, struct squid_radix_node_head *head) {
-    caddr_t v = v_arg;
+    char *v = v_arg;
     register struct squid_radix_node *t = head->rnh_treetop, *x;
-    register caddr_t cp = v, cp2;
-    caddr_t cplim;
+    register char *cp = v, *cp2;
```

Comment:
AFAIK the register modifier is ignored by now; will it make sense to remove it while we're here?

Reply:
IMO best to do that in a separate PR.